unit/src/main/java/org/apache/falcon/unit/LocalExtensionManager.java

```diff
@@ -88,7 +104,19 @@ public APIResult submitAndSchedulableExtensionJob(String extensionName, String j
         return new APIResult(APIResult.Status.SUCCEEDED, "Extension job submitted successfully" + jobName);
```

Comment:
may be I have commented at wrong place, but messages for both submit and submitAndSchedule was same.